core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -308,6 +307,18 @@ private String formatErrorMessage(String format, Object... args) {
         + String.format(" (%s)", mDescription);
   }
 
+  /**
+   * Utility interface for streaming rpc lambda.
+   *
+   * @param <ReqT> Rpc Request Type
+   * @param <ResT> Rpc Response Type
+   */
+  @FunctionalInterface
+  public interface StreamRpcFunction<ReqT, ResT> {
+    @SuppressWarnings("checkstyle:MissingJavadocMethod")
+    StreamObserver<ReqT> apply(StreamObserver<ResT> t) throws IOException;
+  }
+
```

Comment:
Since our `BlockWorkerClient` methods throw exception, we can't use java `Function`, so we have to create a functional interface here